e2e/process-services-cloud/process-task-attach-content-file-cloud.e2e.ts

```diff
@@ -0,0 +1,160 @@
+    });
+
+    it('[C311290] Should be able to attach multiple files when widget allows multiple files to be attached from content', async () => {
+        await appListCloudComponent.checkAppIsDisplayed(simpleApp);
+        await appListCloudComponent.goToApp(simpleApp);
+        await processCloudDemoPage.openNewProcessForm();
+        await startProcessPage.startProcessWithProcessDefinition(processName, 'upload-single-multiple-pro');
+
+        await processCloudDemoPage.processFilterCloudComponent.clickOnProcessFilters();
+        await processCloudDemoPage.processFilterCloudComponent.clickRunningProcessesFilter();
+        await expect(await processCloudDemoPage.processFilterCloudComponent.getActiveFilterName()).toBe('Running Processes');
+        await processCloudDemoPage.processListCloudComponent().checkContentIsDisplayedByName(processName);
+        await processCloudDemoPage.processListCloudComponent().selectRow(processName);
+
+        await tasksCloudDemoPage.taskListCloudComponent().checkTaskListIsLoaded();
+        await tasksCloudDemoPage.taskListCloudComponent().selectRow('UploadSingleMultipleFiles');
+
+        await taskFormCloudComponent.formFields().checkFormIsDisplayed();
+        await taskFormCloudComponent.formFields().checkWidgetIsVisible(widgets.contentMultipleAttachFileId);
+        await attachFileFromContent(pdfFileOne.name);
+        await viewAttachedFile(pdfFileOne.name);
+
+        await taskFormCloudComponent.formFields().checkWidgetIsVisible(widgets.contentMultipleAttachFileId);
+
+        await attachFileFromContent(pdfFileTwo.name);
+        await viewAttachedFile(pdfFileTwo.name);
+        await taskFormCloudComponent.clickCompleteButton();
+
+        await expect(await tasksCloudDemoPage.taskFilterCloudComponent.getActiveFilterName()).toBe('My Tasks');
+        await tasksCloudDemoPage.taskListCloudComponent().checkContentIsNotDisplayedByName('UploadSingleMultipleFiles');
+
+        await tasksCloudDemoPage.taskFilterCloudComponent.clickCompletedTasksFilter();
+        await tasksCloudDemoPage.taskListCloudComponent().checkContentIsDisplayedByName('UploadSingleMultipleFiles');
+
+        await processCloudDemoPage.processFilterCloudComponent.clickOnProcessFilters();
+        await processCloudDemoPage.processFilterCloudComponent.clickCompletedProcessesFilter();
+
+        await expect(await processCloudDemoPage.processFilterCloudComponent.getActiveFilterName()).toBe('Completed Processes');
+        await processCloudDemoPage.processListCloudComponent().checkContentIsDisplayedByName(processName);
+    });
+
+    async function attachFileFromContent(fileName: string) {
+        const contentUploadFileWidget = await processCloudWidget.attachFileWidgetCloud(widgets.contentMultipleAttachFileId);
+        await contentUploadFileWidget.clickAttachContentFile(widgets.contentMultipleAttachFileId);
+
+        await contentNodeSelectorDialog.checkDialogIsDisplayed();
```

Comment:
Thinking that this method could be useful for some other tests, i would recommend to move it[lines 127-144] under contentNodeSelectorDialog PO.

Reply:
Added as part of contentNodeSelectorDialog page. Thanks